src/compas_fab/robots/planning_scene.py

```diff
@@ -59,6 +74,19 @@ def scale(self, transformation):
 class AttachedCollisionMesh(object):
     """Represents a collision mesh that is attached to a :class:`Robot`'s :class:`~compas.robots.Link`.
 
+    Parameters
+    ----------
+    collision_mesh : :class:`compas_fab.robots.CollisionMesh`
+        The collision mesh we want to attach.
```

Comment:
Try to avoid first persons.

```suggestion
        The collision mesh to be attached to the robot model.
```